mmocr/utils/ocr.py

```diff
@@ -478,7 +601,11 @@ def single_pp(self, result, model):
             if export:
                 mmcv.dump(res, export, indent=4)
             if output or self.args.imshow:
-                res_img = model.show_result(arr, res, out_file=output)
+                if model in ['Tesseract_det', 'Tesseract_recog']:
+                    res_img = TextDetectorMixin(show_score=False).show_result(
```

Comment:
Detector and recognizer uses different visualization methods. For recognizers, we use [`BaseRecognizer.show_result()`](https://github.com/open-mmlab/mmocr/blob/75d32504e002f7da3c38c04babd80182be836339/mmocr/models/textrecog/recognizer/base.py#L181).